packages/bpk-component-chip-group/src/BpkChipGroup-test.tsx

```diff
@@ -0,0 +1,247 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import BpkChipGroup, { BpkChipGroupState, CHIP_COMPONENT, CHIP_GROUP_TYPES } from './BpkChipGroup';
+
+const defaultProps = {
+  type: CHIP_GROUP_TYPES.wrap,
+  ariaLabel: 'a11y label',
+}
+
+describe('BpkChipGroup', () => {
+  beforeEach(() => {
+    window.matchMedia = jest.fn().mockImplementation(() => ({
+      matches: true,
+      addEventListener: jest.fn(),
+      removeEventListener: jest.fn(),
+    }));
+  });
+
+  const chips = [
+    {
+      text: 'London',
+    },
+    {
+      text: 'Berlin',
+      selected: true,
+    },
+    {
+      text: 'Florence',
+    },
+    {
+      text: 'Stockholm',
+    }
+  ];
+
+  it('should render correctly with type = rail', () => {
+    const { asFragment } = render(<BpkChipGroup {...defaultProps} chips={chips} type={CHIP_GROUP_TYPES.rail} />);
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
Snapshots are known to not always be reliable or good for testing, can we make a switch out from this to do some other render check for reliability?

Reply:
Makes sense, I have now refactored them 👍